srcs/app/server/Connection.cpp

```diff
@@ -8,29 +8,29 @@ Connection::Connection(
 					response_factory_(response_factory),
 					request_(request),
 					response_(NULL),
-					keep_alive_(true) {}
+					keep_alive_(true),
+					is_cgi_(false),
+					cgi_output_fd_(-1) {}
```

Comment:
Not using finally `fd_set` container :cry: ?

Reply:
What do you mean?

Reply:
Sorry, I though I made a comment above :s 

What I mean is that `FDsets` class was added in order to encapsulate addition/deletion and having a common `fd_set`. Right now, how this is implemented for `CGI` is lifting up the `fd` of the socket from `Connection` to `Server`  and to `Webserver` with methods like:

```c++
int	Connection::GetCgiOutputFd() const {
	return response_->GetCgiOutputFd();
}
```
and

```c++
int	Server::GetCgiOutputFd(int sd) {
	return connections_[sd]->GetCgiOutputFd();
}
```

Does it make sense to have a common `fd_set` and then, lifting up `fd` for this particular case? Wouldn't be easier to just inject `FDsets` instance which is already in `Server` class?

Reply:
If we inject the `FDsets`, we would also have to inject the cgi handlers maps (at least the way it functions now).